source/game_sa/PlantMgr.cpp

```diff
@@ -89,7 +89,7 @@ void CPlantMgr::InjectHooks() {
     RH_ScopedInstall(_ProcessEntryCollisionDataSections_AddLocTris, 0x5DC8B0);
     RH_ScopedInstall(_ProcessEntryCollisionDataSections_RemoveLocTris, 0x5DBF20);
     RH_ScopedInstall(_UpdateLocTris, 0x5DCF00);
-    RH_ScopedGlobalInstall(LoadModels, 0x5DD220);
+    //RH_ScopedGlobalInstall(LoadModels, 0x5DD220); // uses `__usecall`, can't hook
```

Comment:
```suggestion
    //RH_ScopedGlobalInstall(LoadModels, 0x5DD220); // uses `__usercall`, can't hook
```

confused me for a second

Reply:
haha, my bad